recipes/hidapi/all/conanfile.py

```diff
@@ -0,0 +1,126 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, MSBuild, tools
+import os, glob
+
+class HidapiConan(ConanFile):
+    name = "hidapi"
+    license = "BSD-Style"
```

Comment:
Yikes this is complicated https://github.com/libusb/hidapi/blob/master/LICENSE.txt

https://spdx.org/licenses/BSD-3-Clause.html https://spdx.org/licenses/GPL-3.0-or-later

```suggestion
    license = "GPL-3.0-or-later", "BSD-3-Clause", "HIDAPI"
```